torch/testing/_internal/common_methods_invocations.py

```diff
@@ -180,7 +183,7 @@ def __init__(self,
                                                # inside of DifferentiableGraphs when this operation is autodiffed.
                                                # Ex: ['aten::add', 'aten::mm'], defaults to an empty list
                                                # Note: currently no ops use fusible nodes
-                 output_func=lambda x: x,  # fn mapping output to part that should be gradcheck'ed
+                 grad_func=lambda op, *args, **kwargs: op(*args, **kwargs),  # wrapper function for gradcheck
```

Comment:
What about making this None and if None it just points to the op?

Reply:
It has to take op has its first argument so it can wrap the op variants as well. I gave it a default lambda as a way to document its use.

Reply:
OK.